set.mm

```diff
@@ -216269,9 +216312,10 @@ net proof size (existence part)? $)
   $}
 
   ${
-    latidm.b $e |- B = ( Base ` K ) $.
-    latidm.j $e |- .\/ = ( join ` K ) $.
-    $( Lattice join is idempotent.  (Contributed by NM, 8-Oct-2011.) $)
+    latjidm.b $e |- B = ( Base ` K ) $.
+    latjidm.j $e |- .\/ = ( join ` K ) $.
+    $( Lattice join is idempotent.  ( ~ unidm analog.)  See ~ posjidm for a
```

Comment:
```suggestion
    $( Lattice join is idempotent.  Analogue of ~ unidm analog.  See ~ posjidm for a
```

Reply:
Same below. Also, remove the "stronger statement" sentence here and below, and do another PR to move those statements from your mathbox to Main.

Reply:
I could potentially do that but the dependency chain is quite long… Moving those might take a while. Maybe revise it to “see also ~ xxx “ before posjidm and posmidm moved to main?

Also, I think some of the theorems indirectly referenced by these two might depend on dual so #4253 should be handled first.

Reply:
I cannot look at the details. In short: you can make comments in your mathbox referencing Main, but the opposite is less common. There are many such examples in contributors' mathboxes.

Reply:
https://github.com/metamath/set.mm/pull/4248/commits/c4a6d893b61d825172d83ffd618ac8a6ff06fac5